web/src/features/App/api/getLicense.ts

```diff
@@ -1,9 +1,15 @@
 // This hook has not been integrated yet.
 import { useQuery } from "react-query";
 import { Utilities } from "../../../utilities/utilities";
+import { useSelectedApp } from "@features/App";
 import axios from "axios";
+import { AppLicense } from "@types";
 
-export const getLicense = async (appSlug: string) => {
+export const getLicense = async ({
+  appSlug,
+}: {
+  appSlug: string;
+}): Promise<{ license: AppLicense } | null | void> => {
```

Comment:
Add return type to fetching function. I need to do this on all of our fetching stuff. Added to the FE [doc](https://app.shortcut.com/replicated/write/IkRvYyI6I3V1aWQgIjYzMjRkZDFjLTg0OWUtNDE1NS04YWMyLTU2MmU1ZjI3MWVkMyI=)